src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/integration/OpenApiDocsIntTest.kt

```diff
@@ -0,0 +1,78 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.integration
+
+import io.swagger.v3.parser.OpenAPIV3Parser
+import net.minidev.json.JSONArray
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.web.server.LocalServerPort
+import org.springframework.http.MediaType
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+class OpenApiDocsIntTest : IntegrationTestBase() {
+  @LocalServerPort
+  private var port: Int = 0
+
+  @Test
+  fun `open api docs are available`() {
+    webTestClient.get()
+      .uri("/swagger-ui/index.html?configUrl=/v3/api-docs")
+      .accept(MediaType.APPLICATION_JSON)
+      .exchange()
+      .expectStatus().isOk
+  }
+
+  @Test
+  fun `open api docs redirect to correct page`() {
+    webTestClient.get()
+      .uri("/swagger-ui.html")
+      .accept(MediaType.APPLICATION_JSON)
+      .exchange()
+      .expectStatus().is3xxRedirection
+      .expectHeader().value("Location") { it.contains("/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config") }
+  }
+
+  @Test
+  fun `the swagger json is valid`() {
+    webTestClient.get()
+      .uri("/v3/api-docs")
+      .accept(MediaType.APPLICATION_JSON)
+      .exchange()
+      .expectStatus().isOk
+      .expectBody().jsonPath("messages").doesNotExist()
+  }
```

Comment:
I don't think this test does anything... and that you only get `messages` once processed by the OpenAPIV3Parser (see test below)?